quick-start/src/test/java/com/marklogic/quickstart/integrationtests/EndToEndAPITest.java

```diff
@@ -126,22 +126,21 @@ public void modelMappingE2ETests() throws JSONException, IOException, Interrupte
 		
 		assertEquals(200, projectInitResponse.statusCode());
 		assertTrue(projectInitStatus);
-		assertTrue(projectPath.contains("marklog-data-hub-develop/marklogic-data-hub/quick-start/ye-olde-project"));
-		System.out.println("projectInitResponse: "+ projectInitResponse.asString());
-		System.out.println("Project Init Test Passed");
+		assertTrue(projectPath.contains("marklogic-data-hub/quick-start/ye-olde-project"));
```

Comment:
unfortunately this assertion will still fail on windows where the path contains backslashes.
You need a system-independent assertion.  You can do this with a regex, by broadening the assertion, or by replacing backslashes with slashes before comparing.